### DISCUSSION TITLE
Saved page is broken/incomplete

### DISCUSSION THREAD

**Discussion by fa1rid** [gildas-lormeau/SingleFile#1376] [Q&A]
Page: https://www.ui.com/edgemax/comparison/

Expected:

![image](https://github.com/gildas-lormeau/SingleFile/assets/6115961/f45bb7e0-20f2-4617-90fe-362815b8a0eb)

Saved:

![image](https://github.com/gildas-lormeau/SingleFile/assets/6115961/a9c39df0-f307-446b-8b55-a7c5c1ae83e8)

**Environment**
 - OS: Win10 Pro
 - Browser: Chrome 119

**Comment by gildas-lormeau** [comment]
I cannot reproduce the issue with the default settings. Did you change any option in SingleFile?

**Comment by gildas-lormeau** [comment]
I'm closing the issue because I was not able to reproduce it. Feel free to give more info if needed.

**Comment by fa1rid** [comment]
> I'm closing the issue because I was not able to reproduce it. Feel free to give more info if needed.

Thanks for your reply. It turned out the issue was from setting "Site access" to "On Click" instead of "On all sites"


![image](https://github.com/gildas-lormeau/SingleFile/assets/6115961/53eef0f4-9ab9-4f53-a2d0-531b9fcc4fea)
